rpcs3/util/sysinfo.cpp

```diff
@@ -342,6 +354,13 @@ std::string utils::get_OS_version()
 	fmt::append(output,
 		"Operating system: Windows, Major: %lu, Minor: %lu, Build: %u, Service Pack: %s, Compatibility mode: %llu",
 		version_major, version_minor, build, has_sp ? holder.data() : "none", compatibility_mode);
+#elif defined (__APPLE__)
+	const int majorVersion = Darwin_Version::getNSmajorVersion();
+	const int minorVersion = Darwin_Version::getNSminorVersion();
+	const int patchVersion = Darwin_Version::getNSpatchVersion();
```

Comment:
we generally use snake_case. (at least for new code)
So majorVersion should be major_version etc.